### PR TITLE
Bindu, Alekhya | MOBN-422 | Add translation messages for updating met…

### PR DESCRIPTION
**Pull request by binduak**
…adata in recurring appointments

### CODE REVIEW

File: ui/app/appointments/controllers/manage/appointmentsCreateController.js [hunk 1]
```diff
@@ -617,8 +617,13 @@ angular.module('bahmni.appointments')
                 }));
             };
 
-            var saveAppointmentSuccessHandler = function () {
-                messagingService.showMessage('info', 'APPOINTMENT_SAVE_SUCCESS');
+            var saveAppointmentSuccessHandler = function (isUpdate) {
+                if ($scope.isRecurringAppointment && isUpdate) {
+                    $scope.appointment.recurrenceInstance === 'allAppointments' ? messagingService.showMessage('info', RECURRING_APPOINTMENT_SAVE_SUCCESS_APPLYFORALL) : messagingService.showMessage('info', 'RECURRING_APPOINTMENT_SAVE_SUCCESS');
```
Comment (vinay-ThoughtWorks) [inline ui/app/appointments/controllers/manage/appointmentsCreateController.js]: Why `RECURRING_APPOINTMENT_SAVE_SUCCESS_APPLYFORALL` is not quoted and else all are quoted?

It seems to be a mistake..! Please verify.

Comment (vinay-ThoughtWorks) [reply ui/app/appointments/controllers/manage/appointmentsCreateController.js]: Instead making the line much large to reed, you assign the value to a variable and finally pass it to `messagingService.showMessage`

Comment (AlekhyaYalla) [reply ui/app/appointments/controllers/manage/appointmentsCreateController.js]: Modified. Done

File: ui/app/appointments/controllers/manage/appointmentsCreateController.js [hunk 2]
```diff
@@ -617,8 +617,14 @@ angular.module('bahmni.appointments')
                 }));
             };
 
-            var saveAppointmentSuccessHandler = function () {
-                messagingService.showMessage('info', 'APPOINTMENT_SAVE_SUCCESS');
+            var saveAppointmentSuccessHandler = function (isUpdate) {
+                if ($scope.isRecurringAppointment && isUpdate) {
+                    const message = $scope.appointment.recurrenceInstance === 'allAppointments' ? 'RECURRING_APPOINTMENT_SAVE_SUCCESS_APPLYFORALL' : 'RECURRING_APPOINTMENT_SAVE_SUCCESS';
```
Comment (sowmika148) [inline ui/app/appointments/controllers/manage/appointmentsCreateController.js]: Use some const or Can we use isUpdate?

Comment (sowmika148) [reply ui/app/appointments/controllers/manage/appointmentsCreateController.js]: Should be done in separate commit.

**Pull request merged**